hasura/graphql-engine
# add pause in event_triggers, fix #3423

### Description



### Changelog


### Affected components


- [ X ] Server
- [  X ] Docs
- [ X ] Tests

### Related Issues



#3423 

### Solution and Design


Added a new configuration field `pause` in the `hdb_catalog.event_triggers` table and a new field `paused` in the `hdb_catalog.event_log` table. Whenever a new event log is created the `paused` field will be equal to that of `pause` field of the event trigger.

### Steps to test and verify


Create a new event trigger with `pause` as true and check that there are no calls made to the webhook provided


#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [ X ] Yes

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
- [ X ] No new GraphQL schema is generated
   
   

#### Breaking changes

- [ X ] No Breaking changes

  1. Metadata API

     Existing `query` types:
     

  2. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  3. Logging
